src/driver/amdxdna/aie2_pci.c

```diff
@@ -273,6 +291,17 @@ static int aie2_mgmt_fw_init(struct amdxdna_dev_hdl *ndev)
 		return ret;
 	}
 
+	ret = aie2_enable_frame_boundary_preempt(ndev, true);
+	if (ret) {
+		XDNA_ERR(ndev->xdna, "Failed to %s fine grain preemption",
+			 disable_fine_preemption ? "disable" : "enable");
+		return ret;
+	}
+
+	/**
+	 * TODO: Fair scheduling feature is a Strix only feature. Cleanup the below implementation
+	 * once we have a generation specific management firmware initialization.
+	 */
```

Comment:
" /** " is special for kernel-doc tool. It is a hint that this is part of the linux document. Only use /** when you know what you are doing.

 ```suggestion
	/*
	 * TODO: Fair scheduling feature is a Strix only feature. Cleanup the below implementation
	 * once we have a generation specific management firmware initialization.
	 */
```